DataScience/test.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Are we going to check in this notebook?

Reply:
I don't see reason why not - we do not have proper tests in this repo and probably do not want to start this now since we are trying to deprecate the repo. And this notebook still has some value for testing.